app/dao/notifications_dao.py

```diff
@@ -234,6 +234,13 @@ def get_notification_by_id(notification_id, service_id=None, _raise=False):
     return query.one() if _raise else query.first()
 
 
+def dao_get_notification_or_history_by_id(notification_id):
```

Comment:
when I saw the method (with _dao_get_ prefix) I wasn't sure if it was returning a dao or an entity. it is now clearer that it returns a notification object and I wonder what is the distinction between _get__ and _dao_get__

Reply:
this is a nomenclature that it seems to be existing within the code

so if we separate the layers of the application we would get something like
handler (endpoint entry where validation occurs)
service (business logic)
dao (database operations)

so because this function is part of a database operation, the prefix dao seems to be what is done in general.
I might be wrong, but that is what I have observed.